tareas/clase-5/tareas-clase-5/tarea-4/tarea-4.js

```diff
@@ -0,0 +1,67 @@
+document.querySelector('#calcular-promedio').onclick = function () {
+    const numerosEnListaLi = document.querySelectorAll('.numero');
+
+    let listaDeNumeros = [];
+    for (let i = 0; i < numerosEnListaLi.length; i++) {
+        listaDeNumeros.push(Number(numerosEnListaLi[i].innerText));
+    }
+    function sumarNumeros(numeros) {
+        let sumaTotal = 0;
+        for (let i = 0; i < numeros.length; i++) {
+            sumaTotal = sumaTotal + numeros[i];
+        }
+        return sumaTotal;
+    }
+    let numeros = listaDeNumeros;
+    const sumaNumerosTotal = sumarNumeros(numeros);
+    document.querySelector(
+        '.suma-numeros-total',
+    ).innerText = `La suma total de todos los numeros es: ${sumaNumerosTotal}`;
+    
+    
+    function calcularPromedio(listaDeNumeros) {
+        let resultado = 0;
+        let promedio = 0;
+        for (let i = 0; i < listaDeNumeros.length; i++) {
+            resultado = resultado + listaDeNumeros[i];
+            promedio = resultado / listaDeNumeros.length;
```

Comment:
este cálculo se hace fuera del for, una vez que sumaste todos los números sólo tenés que dividir 1 sola vez, no CADA VEZ